### PR TITLE
L2 345 style scrollbar

### PR DESCRIPTION
**Pull request by wackyleo459**
Style of scrollbar will have dark theme so that anytime a scrollbar will be shown, the default style is shown.
Also light modal will have light theme scrollbar.
Screen Shot 2022-05-06 at 3 02 53 PM
Screen Shot 2022-05-06 at 3 01 07 PM

### CODE REVIEW

File: frontend/svelte/src/lib/themes/theme.scss [hunk 1]
```diff
@@ -22,6 +22,21 @@ body {
   box-sizing: border-box;
 }
 
+::-webkit-scrollbar {
+  background: #6c6e6e;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/theme.scss:26]: shouldn't we rather use colors from our theme - i.e. `variables.scss`?

File: frontend/svelte/src/lib/themes/theme.scss [hunk 2]
```diff
@@ -22,6 +22,21 @@ body {
...
+}
+
+::-webkit-scrollbar-thumb {
+  background: #48494c;
+  height: 200px;
+  border: solid 2px #6c6e6e;
+  -webkit-border-radius: 1ex;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/theme.scss:33]: I never used `ex` actually, any particular reason to use such unit?

Comment (wackyleo459) [reply frontend/svelte/src/lib/themes/theme.scss:33]: Honestly, it's my first time. `ex` is half the size of `em` (or essentially height of an x) for vertical height. What I can do is change to 0.5 em for consistency. It literally is the same thing.

File: frontend/svelte/package-lock.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
       "dependencies": {
         "@dfinity/agent": "^0.11.0",
         "@dfinity/auth-client": "^0.11.0",
-        "@dfinity/nns": "nightly",
+        "@dfinity/nns": "^0.4.0-nightly-2022-05-06",
```
Comment (peterpeterparker) [inline frontend/svelte/package-lock.json:13]: can you revert the changes in `package-lock.json` from this PR please

File: frontend/svelte/src/lib/themes/theme.scss [hunk 4]
```diff
@@ -22,6 +22,37 @@ body {
...
+  border: solid 2.5px var(--gray-600);
+  border-radius: 0.5em;
+  -webkit-border-radius: 0.5em;
+}
+
+::-webkit-scrollbar-corner {
+  background: var(--gray-600);
+}
+
+.light > div.wrapper {
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/theme.scss:41]: should we move this rule to the `Modal.svelte`?

File: frontend/svelte/src/lib/themes/theme.scss [hunk 5]
```diff
@@ -22,6 +22,37 @@ body {
...
+  ::-webkit-scrollbar {
+    background: var(--gray-50er-background-contrast);
+  }
+  ::-webkit-scrollbar-thumb {
+    background: var(--light-background-shade);
+    border: solid 2.5px var(--gray-50er-background-contrast);
+    border-radius: 0.5em;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/theme.scss:48]: these two rules are already set, I think you don't have to declare them again

File: frontend/svelte/src/lib/themes/theme.scss [hunk 6]
```diff
@@ -22,6 +22,37 @@ body {
...
+  background: var(--gray-600);
+}
+
+.light > div.wrapper {
+  ::-webkit-scrollbar {
+    background: var(--gray-50er-background-contrast);
+  }
+  ::-webkit-scrollbar-thumb {
+    background: var(--light-background-shade);
+    border: solid 2.5px var(--gray-50er-background-contrast);
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/theme.scss:47]: just in case you would like to go fancy and avoid to redeclare the width, you can use a variable.

```
::-webkit-scrollbar-thumb {
   border: 2.5px solid var(--scrollbar-border-color);
   --scrollbar-border-color: var(--gray-600);
 }
 
 .light > div.wrapper {
   --scrollbar-border-color: var(--gray-50er-background-contrast);
 }

File: frontend/svelte/src/lib/themes/theme.scss (hunk 4 above)
Comment (wackyleo459) [reply frontend/svelte/src/lib/themes/theme.scss:41]: I've moved line 41 (.light > div.wrapper ) into Modal. But lines 25-39 are kept in current file, since dark scrollbar applies to body page as well.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/themes/theme.scss:41]: Cool!

Did you push your changes? PR still look the same.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks
